data/outport.go

```diff
@@ -57,10 +60,38 @@ type BlockScrs struct {
 
 // BlockEventsWithOrder holds the block transactions with order
 type BlockEventsWithOrder struct {
-	Hash      string                                `json:"hash"`
-	ShardID   uint32                                `json:"shardID"`
-	TimeStamp uint64                                `json:"timestamp"`
-	Txs       map[string]TransactionWrapped         `json:"txs"`
-	Scrs      map[string]SmartContractResultWrapped `json:"scrs"`
-	Events    []Event                               `json:"events"`
+	Hash      string                                     `json:"hash"`
+	ShardID   uint32                                     `json:"shardID"`
+	TimeStamp uint64                                     `json:"timestamp"`
+	Txs       map[string]*InterceptorTransaction         `json:"txs"`
+	Scrs      map[string]*InterceptorSmartContractResult `json:"scrs"`
+	Events    []Event                                    `json:"events"`
+}
+
+// InterceptorTransaction defines a wrapper over transaction
+type InterceptorTransaction struct {
```

Comment:
Not quite happy with this name chosen. "Interceptor" 
Might rename all those transactions structures?

Reply:
done